### PR TITLE
Health

### PR DESCRIPTION
**Pull request by mayurbhandary**
Adds connection count to health endpoint and adds PR template.

### CODE REVIEW

**Comment by natenichols** [comment]
Any particular reason why you want the number of connections to persist? Why not just add a function that returns `connections.length` in `connections.ts`, and report that in your health endpoint?

**Comment by natenichols** [comment]
Ohhh, is see, cuz you want to report it on the `./api` directory.

Yeah, I'm cool w/ persisting, will start a review

File: src/shared/database/index.ts [hunk 1]
```diff
@@ -72,6 +72,7 @@ async function setupCrawlsTable(): Promise<void> {
       table.string('ip')
       table.integer('port')
       table.string('ws_url')
+      table.boolean('connected')
```
Comment (natenichols) [inline src/shared/database/index.ts:75]: This requires an alteration to the database. We'll want to be certain that we record the process for deploying to stage, and be careful deploying to prod.

Comment (natenichols) [reply src/shared/database/index.ts:75]: https://www.postgresql.org/docs/12/sql-altertable.html

**Review by natenichols — APPROVED** [review]
So, I'm approving, because this LGTM. 

Would you post an example request/response on your local branch to this thread, so we can see what this health check looks like.

**Comment by mayurbhandary** [comment]
@natenichols, example response:
`{"count":"48"}`

**Pull request merged**